test/langtools/tools/javac/StringConcat/TestIndyStringConcat.java

```diff
@@ -34,23 +34,23 @@
  * @summary Test that StringConcat is working for JDK >= 9
  * @modules jdk.jdeps/com.sun.tools.classfile
  *
- * @clean TestIndyStringConcat*
+ * @clean TestIndyStringConcat
```

Comment:
While this class does not appear to generate any nested classes, you may still want to use `@clean *` to ensure that any/all previously generated class files are removed.

Reply:
Right, fixed.